packages/cli/test/fixtures/discover/mem.datasource.js.txt

```diff
@@ -0,0 +1,106 @@
+const DataSource = require('loopback-datasource-juggler').DataSource;
+
+const modelList = [
+  {
+    name: 'Test',
+    view: false,
+    schema: '',
+  },
+  {
+    name:'Schema',
+    view: false,
+    schema: 'aSchema'
+  },
+  {
+    name:'View',
+    view: true,
+    schema: ''
+  }
+];
+// In real model definitions, the schema is contained in options->connectorName->schema
+const fullDefinitions = [
+  {
+    'name': 'Schema',
+    'schema': 'aSchema',
+    'properties': {}
+  },
+  {
+    'name': 'View',
+    'view': true,
+    'schema': '',
+    'properties': {}
+  },
+  {
+    'name': 'Test',
+    'properties': {
+      'dateTest': {
+        'type': 'Date',
+        'required': false,
+        'length': null,
+        'precision': null,
+        'scale': null,
+      },
+      'numberTest': {
+        'type': 'Number',
+        'required': false,
+        'length': null,
+        'precision': null,
+        'scale': null,
+      },
+      'stringTest': {
+        'type': 'String',
+        'required': false,
+        'length': null,
+        'precision': null,
+        'scale': null,
+      },
+      'booleanText': {
+        'type': 'Boolean',
+        'required': false,
+        'length': null,
+        'precision': null,
+        'scale': null,
+      },
+      'id': {
+        'type': 'Number',
+        'required': true,
+        'length': null,
+        'precision': null,
+        'scale': 0,
+        'id': 1,
+      },
+    },
+  },
+];
+
+// TODO add .txt to this extension so it doesn't get called by global tests`
```

Comment:
This TODO does not seem to be relevant any more, the file already has `.txt` extension.